dynamicsamplingprocessor/testdata/config.yaml

```diff
@@ -0,0 +1,22 @@
+processors:
+  dynamic_sampler:
+    # option one: EMADynamicSampler
+    - sampler: "EMADynamicSampler"
+      goal_sample_rate: 10
+      key_fields: ["key1", "key2"]
+
+    # option two: EMAThroughputSampler
+    # name: "EMAThroughputSampler"
+    # # goal_throughput_per_second is the target number of events to send per second.
+    # goal_throughput_per_second: 100
+    # key_fields: ["key1", "key2"]
```

Comment:
We might wanna bikeshed a little bit on the format of the config here, and if we support the full range of samplers or not. We'll probably also want to document which should _generally_ be the best for log data as it flows through one event at a time.

Reply:
Yeah, agree. This was mostly to put this in a place so improve visibility and we can work on it together.